### PR TITLE
Use absolute symlinks if given a relative path

### PR DESCRIPTION
**Pull request by CampGareth**
Currently `reco check --source addition` breaks when run from `/examples` as this results in a symlink from our virtual gopath to `/examples/vendor` which doesn't exist. With this PR the symlink will now point to `/examples/addition/vendor` which does exist and `reco check` will work as intended.

### CODE REVIEW

**Comment by CampGareth** [comment]
Appveyor is failing due to unrelated chocolatey errors.

File: cmd/check.go [hunk 1]
```diff
@@ -67,8 +67,14 @@ func check(_ *cobra.Command, args []string) {
 		exitWithError(fmt.Errorf("%s not found", srcFile))
 	}
 
+	//reco-check expects absolute paths so convert relative to absolute here
+	absSrcFile, err := filepath.Abs(srcFile)
+	fmt.Println(absSrcFile)
```
Comment (joshbohde) [inline cmd/check.go]: Remove this.

Comment (CampGareth) [reply cmd/check.go]: oops, didn't spot that fmt

**Comment by CampGareth** [comment]
[Reco check --source does not work for relative paths](https://trello.com/c/zh5YlaCN/255-reco-check-source-does-not-work-for-relative-paths)

**Pull request merged**